bpf/process/bpf_execve_event.c

```diff
@@ -252,6 +258,8 @@ execve_send(struct sched_execve_args *ctx)
 #endif
 	}
 
+	execve_joined_info_map_clear(tid);
```

Comment:
Do we really need this?  I don't see a code path where `execve_joined_info_map_get` is not called but this function is. I would suggest to to keep things simple and remove `execve_joined_info_map_clear`.

Reply:
Done, thanks @kkourt